### PR TITLE
Remove unused historic files and code.

### PR DESCRIPTION
**Pull request by mikebryant**
They can all be retrieved from the history - better not to have them cluttering up the repo.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -32,7 +30,7 @@ To get good API documentation, run:
 
     $ epydoc -n "PyFCP API manual" -o html fcp
 
-When you install this package (refer INSTALL), you should 
+When you install this package (refer INSTALL), you should
```
Comment (Bombe) [inline README]: I’m not a friend of how you mix whitespace changes and unrelated file removals in the same commit.

Comment (mikebryant) [inline README]: Better?

Comment (Bombe) [inline README]: I’m not opposed to the whitespace changes themselves. Just put them into a separate commit and it’s fine with me. :)

Comment (mikebryant) [inline README]: Fair enough. Will do it later though, in a more structured fashion - wanted to get this pull request in a viable state first.

**Comment by Thynix** [comment]
Thanks! In the future please note that the project [commit message standards](https://wiki.freenetproject.org/Coding_standards#Commit_Messages) end the summary line without a full stop. It's not a big deal so I've merged it anyway.
